recipes/diligent-core/all/conanfile.py

```diff
@@ -154,8 +154,17 @@ def package_info(self):
 
         self.cpp_info.libs = tools.collect_libs(self)
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore"))
-        # fake target. Needed for DiligentFx to handle paths like ../../../DiligentCore
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Common", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Primitives", "interface"))
```

Comment:
wow, that's a lot new includes

Reply:
It's weird. It brings many generic headers names without namespace folder.

Reply:
Yes, it's really confusing...
But look for example on https://github.com/DiligentGraphics/DiligentTools/blob/master/TextureLoader/include/pch.h

These files are referenced without full path...